pytravharv/__main__.py

```diff
@@ -45,119 +43,49 @@ def get_arg_parser():
         "-n",
         "--name",
         type=str,
+        required=False,
         default=None,
         help="Name of the configuration to use",
     )
 
+    parser.add_argument(
+        "-m",
+        "--mode",
+        choices=["memory", "uristore"],
+        default="memory",
+        required=True,
+        help="Mode to use, either memory or uristore. Default is memory. If memory is used, the target store will be a temporary in-memory store. If uristore is used, the target store will be a URI store.",
+    )
+
     parser.add_argument(
         "-o",
```

Comment:
feels more like `-d` `--dump`

as it dumps the end-state (== all triples inside) of the 'store' to this file at the end of the process?